packages/dx-react-grid/src/plugins/virtual-table/virtual-table-state.tsx

```diff
@@ -116,10 +118,18 @@ class VirtualTableStateBase extends React.PureComponent<VirtualTableStateProps,
   static getDerivedStateFromProps(nextProps, prevState) {
     const {
       availableRowCount = prevState.availableRowCount,
+      totalRowCount,
+      pageSize,
+      infiniteScrolling,
     } = nextProps;
 
     return {
-      availableRowCount,
+      availableRowCount: getAvailableRowCount(
+        infiniteScrolling,
+        pageSize * 2,
+        availableRowCount,
+        totalRowCount,
+      ),
```

Comment:
At line 156 `const availableRowCount = infiniteScrolling ? stateRowCount : totalRowCount;`

Is this necessary code? Seems that the `getAvailableRowCount` function has this statement